crossplane-api/api/a9s/mariadb/definition.yaml

```
@@ -79,6 +79,28 @@ spec:
                       message:
                         "Plan migrations not possible from Non-SSL to SSL and
                         SSL to Non-SSL"
+                  
+                # Custom Parameters
+                parameters:
+                  type: object
+                  properties:
+                    binlogExpireDays:
+                      type: integer
+                      description: |
+                        The number of days that the binary log should be kept on the instance that is running MariaDB before purging.
+                      x-kubernetes-validations:
+                        - rule: "self > 0"
+                          message: "Must be greater than 0" # TODO: test if 0 or 1
```

suggestion:
The CEL rule looks valid. However, for this kind of simple validation, it might be better to use standard OpenAPI v3 validation, which is natively supported by the Kubernetes API and faster:
```suggestion
                      minimum: 1
```